### ISSUE TITLE
Version 2 macOS M1: hotkey for searching expansions collides with my Alfred.

### ISSUE THREAD

**Issue by raguay** [federico-terzi/espanso#878]
**Is your feature request related to a problem? Please describe.**
I use <Opt>-<Space> to launch Alfred, but espanso opens the expansions list (command prompt?). This keeps me from being able to use Alfred. I have other program on <cmd><space> and <ctrl><space> that I don't want to change due to muscle memory.

**Describe the solution you'd like**
I would like to be able to configure the hotkey to use.

**Describe alternatives you've considered**
At least a way to turn off the hotkey for it.

**Additional context**

**Comment by federico-terzi** [comment]
Hey @raguay,

Thanks for reporting this issue! The documentation is still lacking, I'm sorry about that, I'm working hard to get it ready for the new release :)

You can customize the search shortcuts by adding the following to your `$CONFIG/config/default.yml` file and change them as you like:

```yaml
search_trigger: "jkj"
search_shortcut: "ALT+SPACE"
```

The valid values are listed here: https://github.com/federico-terzi/espanso/blob/master/espanso-detect/src/hotkey/keys.rs#L221-L302
Though I haven't tested them all, some of them might be broken :)

Let me know if that helps

Cheers :)

**Comment by raguay** [comment]
Hi,

Thanks. That is just what I needed. Now waiting on the downloading of packages to work.

Richard

**Comment by dcd-arnold** [comment]
Hi,

thank you for `espanso`. I use it for some time now an love it.
I know this issue is closed, however I am on `espanso 2.1.6-beta` (installed with homebrew) on an M1. I tried to configure another shortcut. Neither of these statements did take:
```yaml
search_shortcut: ALT+SHIFT+SPACE
search_shortcut: off
```
Matching works as expected. 

@federico-terzi Should I open another issue?

**Comment by federico-terzi** [comment]
@dcd-arnold I'm sorry you experienced this problem, could you please open another issue and post the output of "espanso log" there? :)

**Comment by dcd-arnold** [comment]
> @dcd-arnold I'm sorry you experienced this problem, could you please open another issue and post the output of "espanso log" there? :)

https://github.com/espanso/espanso/issues/1354
